JavaScript/JS-Week_1/Javascript_Homework/Age-ify/script.js

```diff
@@ -0,0 +1,5 @@
+const yearOfbirth = 1994;
+const yearFuture = 2045;
+let age =  yearFuture - yearOfbirth;
```

Comment:
`age` can also be defined as a `const` as it's not changed later.

Reply:
Thank you for your suggestion. I changed it as you said.